lib/resty/dns/resolver.lua

```diff
@@ -300,109 +308,18 @@ local function _build_request(qname, id, no_recurse, opts)
     }
 end
 
+local function parse_section(answers, section, buf, start_pos, size, should_skip)
```

Comment:
Style: this line exceeds 80 columns. This can be found by the `lua-releng` tool.